test/unit/collection-view/collection-view-viewmixin.spec.js

```diff
@@ -163,11 +163,8 @@ describe('CollectionView - ViewMixin', function() {
       collectionView._removeChildren();
     });
 
-    // Since the collectionView is destroyed we
```

Comment:
in this case this test was added early on when at one point all views were removed in an `_.each`.  This test was added when it was determined that it wasn't necessary.  Now the `_init` in the `_destroyView` does set the length to 0, but does it cheaply